README.md

```diff
@@ -1,24 +1,208 @@
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="https://user-images.githubusercontent.com/84629565/202665566-ba1a8ed3-041f-45bc-b21b-efdcc357189b.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:card_index: Entity Relationship Diagram](#er-diagram)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 Budget App <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+**Budget App** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.The Ruby on Rails application that allows the user to:
 
-* Deployment instructions
+ - register and log in, so that the data is private to them.
+ - introduce new transactions associated with a category.
+ - see the money spent on each category.
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Create and run necessary migration files]**
+- **[Match table and column names to the ERD diagram.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ER DIAGRAM-->
+
+## :card_index: Entity Relationship Diagram <a name="er-diagram"></a>
+
+![er-diagram](https://github.com/microverseinc/curriculum-rails/raw/main/capstone/images/erd_diagram.png)
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://rails-biih.onrender.com)
+
+<!-- PROJECT DOCUMENTATION -->
+## 🚀 Project Documentation <a name="project documentation"></a>
+
+- [Documentation Link](https://www.loom.com/share/bfa26e63901d4654a293ac328ff94a18)
```

Comment:
- [ ] @peterboro great work on your presentation, however, you didn't highlight an interesting piece of code or something you built that you are very proud of, also I noted your voice is way faster than what is displayed and the video cuts at 2 minutes(maybe an issue with loom you might try removing and adding it again ).